jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -242,7 +242,7 @@ apkSignature.unprotectedEntry=Dateien, die nicht durch eine APK-Signatur v1 gesc
 issues_panel.label=Probleme:
 issues_panel.errors=%d Fehler
 issues_panel.warnings=%d Warnungen
-issues_panel.tooltip=Im der Logansicht anzeigen
+issues_panel.tooltip=Im Protokoll-Anzeiger öffnen
```

Comment:
I don't think "Anzeiger" is the correct word here see https://www.duden.de/rechtschreibung/Anzeiger